transforms/universal/html2parquet/python/src/html2parquet_main.py

```diff
@@ -0,0 +1,110 @@
+import time
+from argparse import ArgumentParser, Namespace
+from typing import Any
+import zipfile
+import io
+import trafilatura
+
+import pyarrow as pa
+# from data_processing_ray.runtime.ray import RayTransformLauncher
+# from data_processing_ray.runtime.ray.runtime_configuration import (
+#   RayTransformRuntimeConfiguration,
+# )
+# import data_processing
+# print(dir(data_processing))
+
+
+from data_processing.transform import AbstractTableTransform, AbstractBinaryTransform, TransformConfiguration
+from data_processing.utils import CLIArgumentProvider, get_logger, TransformUtils
+
+
+class HtmlToParquetTransform(AbstractTableTransform):
```

Comment:
You are not doing table transform. plas replace AbstractTableTransform with AbstractBinaryTransform